extensions/sql/data/SQLIdentifier.cpp

```diff
@@ -16,30 +16,26 @@
  * limitations under the License.
  */
 
-#include "Utils.h"
+#include "SQLIdentifier.h"
 
-#include <vector>
-#include <string>
+namespace org::apache::nifi::minifi::sql {
 
-#include "utils/StringUtils.h"
+SQLIdentifier::SQLIdentifier(std::string str) {
```

Comment:
As it is a self-contained entity I would add some simple unit tests for it.

Reply:
added unit tests